tests/zipkin_test.py

```diff
@@ -458,6 +458,24 @@ def test_func(a, b):
     pop_zipkin_attrs_mock.assert_called_once_with()
 
 
+@mock.patch('py_zipkin.zipkin.create_endpoint', wraps=zipkin.create_endpoint)
+def test_zipkin_span_decorator_many(create_endpoint_mock):
```

Comment:
The first divergence happens [here](https://github.com/Yelp/py_zipkin/blob/6676f0fdebcab2a3f980ac3a564d1b29e0029752/py_zipkin/zipkin.py#L207), which means I can use `get_zipkin_attrs` to do the same test.

Maybe the more correct way is to do some mock trickery that allows us to inspect the state of the zipkin_span instance, but I can't think of a clean way to do that right now